src/crypto/store/localStorage-crypto-store.js

```diff
@@ -167,6 +168,57 @@ export default class LocalStorageCryptoStore extends MemoryCryptoStore {
         func(result);
     }
 
+    getSessionsNeedingBackup(limit) {
+        const sessionsNeedingBackup
+              = getJsonItem(this.store, KEY_SESSIONS_NEEDING_BACKUP) || {};
+        const sessions = [];
+
+        for (const session in sessionsNeedingBackup) {
+            if (Object.prototype.hasOwnProperty.call(sessionsNeedingBackup, session)) {
+                const senderKey = session.substr(0, 43);
```

Comment:
the magic numbers here feel scary; is this really the best we can do? at least, can they be const'd?

Reply:
or factored out ,given they appear in the memory-crypto-store too?